ckanext/scheming/templates/scheming/snippets/form_field.html

```diff
@@ -58,6 +60,25 @@
 
     <div name="repeating-template" style="display:none">{{ repeating_panel('REPEATING-INDEX0', 'REPEATING-INDEX1') }}</div>
   </fieldset>
+
+{% block removal_popup %}
+  <div class="modal" id="mid-screen" tabindex="-1" role="dialog" aria-labelledby="mid-screenLabel" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="mid-screenLabel">{{ _('Confirmation of Removal') }}</h5>
+        </div>
+        <div class="modal-body" style="text-align:center;">
+            <p id="statusText0">{{ _('Are you sure you want to remove') }} {{ h.scheming_language_text(field.label) }} </p><p id="statusText1"></p>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-secondary" data-dismiss="modal">{{ _('Cancel') }}</button>
+          <button name="confirmRemoval" id="confirmRemoval" type="button" class="btn btn-primary" data-dismiss="modal">{{ _('Confirm') }}</button>
+        </div>
+      </div>
+    </div>
+  </div>
+{% endblock %}
```

Comment:
The rest of ckan uses confirmation boxes like this: 
https://github.com/ckan/ckan/blob/ckan-2.8.5/ckan/templates/group/snippets/group_form.html#L37
which is much less code to maintain.

Can you see if you can use something like this instead? The text I would use is "Are you sure you want to remove these fields?" because we can't assume it makes sense to stick the field label on the end of a sentence in any language (e.g. it has the wrong capitalization in French). We could allow the user to define a `delete_subfields_confirmation` message in the yaml file to override this text and retrieve it with `h.scheming_language_text`.

Reply:
Are you referring specifically to delete_button_text block of the link you provided? I can implement that. For the removal message, would something like:

> Status 1

> Are you sure you want to remove this?

be ok?

Reply:
I was referring to the `data-module-content="{{ _('Are you sure you want to delete this Group?') }}` example, making it possible to change the confirmation string displayed from the schema.

I like your suggestion but it's not a field it's a group of fields, right? "Group" is overloaded so what do you think of 
> Status 1
> Are you sure you want to remove these values?"